### DISCUSSION TITLE
Does transitland re-contributes data to google maps?

### DISCUSSION THREAD

**Discussion by TheNoim** [transitland/transitland#339] [Q&A] [answered]
Hi,

My name is Nils. Recently I moved from Hannover to Stralsund in Germany. In Hannover I was used to check my bus and train rides with google maps. It all worked flawlessly, because the GVH and Üstra contributes Data to google maps. However, in my new location there is no such integration. It seems like the [vvr](https://www.vvr-bus.de/) isn't interested in contributing data for easier navigation. So I looked up how google gets it data. Then I found out about the [GTFS](https://developers.google.com/transit). But you can't just contribute your own data via some selfhosted server. So, I started to search for different ways to get data into google maps and found transitland. 

tldr: My question is: If someone contributes data from vvr to transitland, would it get imported into google maps? Or is transitland really just a collection of GTFS for people to use? Or do you guys have a different idea how I could archive my goal?

**Comment by drewda** [comment, marked as answer]
Hello Nils,

Transitland is used by a wide range of people and organizations both through the web interface at www.transit.land and through the APIs. However, to the best of our knowledge, Google is not one of them. Google manages their own relationships and integrations with transit agencies.

If VVR is able to provide a GTFS feed, we would be glad to add it to Transitland.

Alternatively, we know some European agencies produce data in other formats, such as NeTEx (Network Timetable EXchange). In that case, the agencies may wish to consider also converting their format to GTFS on a regular basis — that would enable Transitland to consume it.

If you have not already, I would recommend contacting VVR and trying to identify an office or staff member who may be able to provide an open-data set. Best of luck.
